core/src/components/datetime/test/format.spec.ts

```diff
@@ -155,4 +155,17 @@ describe('getLocalizedTime', () => {
 
     expect(getLocalizedTime('en-US', datetimeParts, false)).toEqual('9:40 AM');
   });
+
+  it('should avoid Chromium bug when using 12 hour time in a 24 hour locale', () => {
+    const datetimeParts = {
+      day: 1,
+      month: 1,
+      year: 2022,
+      hour: 0,
+      minute: 0,
+      tzOffset: 0,
+    };
+
+    expect(getLocalizedTime('en-GB', datetimeParts, false)).toEqual('12:00 am');
```

Comment:
The original report was 12pm rather than 12am; does it matter?

Reply:
In this case, the day period does not make a difference. Chromium will still output the 12 hour as "0":

```js
new Intl.DateTimeFormat('en-GB', { hour: 'numeric', minute: 'numeric', hour12: true }).format(new Date('2022-06-01T00:00:00'))
> '0:00 am'
new Intl.DateTimeFormat('en-GB', { hour: 'numeric', minute: 'numeric', hour12: true }).format(new Date('2022-06-01T12:00:00'))
> '0:00 pm'
```